### PR TITLE
[IOPSC-57] Code for change feed handler

### PR DESCRIPTION
**Pull request by LoFrance**
This PR add a handler to retrieve documents from Cosmos change feed, retrieve data from Microsoft APIM, for each document, and create a new dataset to insert in a relational database, in such case a PostgreSQL.

The problems we have to solve are important to our operation team because they needs a set of functions to get some exports and info about our data.

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: OnServiceChange/handler.ts [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+const handler = (): Handler =>
+  pipe(
+    T.of(void 0),
+    T.map(_ => void 0)
+  );
+
+const OnServiceChangeHandler = () => (
+  _context: Context,
+  _documents: ReadonlyArray<unknown>
+): Handler => pipe(handler());
```
Comment (balanza) [inline OnServiceChange/handler.ts:161]:
```suggestion
): Handler => handler();
```

File: OnServiceChange/handler.ts [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+      `"${dbConfig.DB_TABLE}"."serviceVersion" < excluded."serviceVersion"`
+    )
+    .toQuery() as NonEmptyString;
+
+// TO DO: This is the Handler and it's to be implemented!
+const handler = (): Handler =>
+  pipe(
+    T.of(void 0),
+    T.map(_ => void 0)
+  );
```
Comment (balanza) [inline OnServiceChange/handler.ts:156]: It seems the handler does nothing

**Review by balanza — CHANGES_REQUESTED** [review]

**Comment by LoFrance** [comment]
This PR was replaced by https://github.com/pagopa/io-developer-portal-service-data/pull/4

File: OnServiceChange/handler.ts (hunk 2 above)
Comment (LoFrance) [reply OnServiceChange/handler.ts:204]: Yes, thanks, next time it's best to handle it with an exception so no documents be skipped

File: OnServiceChange/handler.ts [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+      // We choose to just skip such documents
+      if (isSubscriptionNotFound(err)) {
+        return TE.of(void 0);
+      } else {
+        return TE.left(err);
+      }
+    }, TE.of)
+  );
+
+// TO DO: This is the Handler and it's to be implemented!
```
Comment (balanza) [inline OnServiceChange/handler.ts:195]:
```suggestion
```

**Review by balanza — APPROVED** [review]

**Pull request merged**